dbaas/psmdb/psmdb.go

```diff
@@ -66,6 +66,17 @@ func (p PSMDB) Name() string {
 }
 
 func (p PSMDB) App() (string, error) {
+	mini, err := dbaas.IsMini()
+	if err != nil {
+		return "", nil
+	}
+	if mini {
+		none := "none"
+		for _, replset := range p.config.Spec.Replsets {
+			replset.Resources = nil
+			replset.MultiAZ.Affinity.TopologyKey = &none
+		}
+	}
```

Comment:
App() is not the best place for the config setup. 
There is a special method for the config creation and setup (for psmdb it's https://github.com/Percona-Lab/percona-dbaas-cli/blob/add-psmdb/dbaas/psmdb/config.go#L513). I'd like keep all such things together, not spread through the code. Just extend signature of these methods with new parameter.

Also, it worth considering to have a "Platform type" instead of `bool`. It would be a more extendible and clear.